content/05_games/00_portal-2.cats

```diff
@@ -8,6 +8,7 @@ Name,Objective,OOB,SLA,"Pause Abuse","Demo Requirement",Leaderboard,Notes,Modera
 "Solo Co-op","Complete courses 1-5 of the cooperative campaign, without skipping any, alone in splitscreen.",Yes,Yes,No,200th,https://www.speedrun.com/portal_2?h=Solo_Co-op&x=prkl4wdn,"`in_forceuser` may be changed, for which `sv_cheats 1` is required. All other command restrictions remain unchanged.","Betsruner, BiSaXa, Fridge"
 "Most Recent Community Chambers","Complete the specified amount of community maps from the Most Recent option.",No,No,No,N/A,https://www.speedrun.com/portal_2_category_extensions?h=Most_Recent_Community_Chambers-10-SP&x=zd363xrd-ylq9vjz8.21gjmx81-p85dqg0l.4qykr321,"Video proof is required. You can skip levels. You cannot play your own maps or maps you have previously seen.","Betsruner, BiSaXa, Fridge, phunkpai"
 "Co-op any% EHM","Complete up to the end of course 5 of the cooperative campaign.",Yes,Yes,No,200th,https://www.speedrun.com/portal_2_category_extensions?h=Co-op_Any_EHM&x=vdononok,"You may use glitches to skip courses and maps. EHM is allowed.","Betsruner, BiSaXa, Fridge, phunkpai"
-"Portal Placement Never Fail","Complete the specified campaign according to the standard ruleset for the underlying category.",Yes,-,-,"All runs",https://www.speedrun.com/portal_2_category_extensions?h=Portal_Placement_Never_Fail-SP&x=wkpo8182-38dw2v1n.8102832q,"You may use `sv_portal_placement_never_fail 1` (which requires `sv_cheats 1`). All other rules come from the underlying category (Inbounds No SLA, AMC, Co-op any%, or Solo Co-op).","Betsruner, BiSaXa, Fridge, phunkpai"
+"Portal Placement Never Fail","Complete the specified campaign according to the standard ruleset for the underlying category.",Yes,No,Yes,"All runs",https://www.speedrun.com/portal_2_category_extensions?h=Portal_Placement_Never_Fail-SP&x=wkpo8182-38dw2v1n.8102832q,"You may use `sv_portal_placement_never_fail 1` (which requires `sv_cheats 1`). All other rules come from the underlying category (Inbounds No SLA, AMC, Co-op any%, or Solo Co-op).","Betsruner, BiSaXa, Fridge, phunkpai"
```

Comment:
These fields are intentionally blanked because these rules are inherited from the underlying category: e.g. you can't pause abuse in coop.